src/routes/v2/authenticatedSites/index.js

```diff
@@ -107,13 +107,15 @@ const getAuthenticatedSitesSubrouter = ({
   })
   const unlinkedPageService = new UnlinkedPageService({ gitHubService })
   const resourcePageService = new ResourcePageService({ gitHubService })
-  const mediaFileService = new MediaFileService({ gitHubService })
+  const mediaFileService = new MediaFileService({ repoService: gitHubService })
```

Comment:
Hmm I think this is not possible yet, because many of the other services expect a `gitHubService` instead of a `repoService`, so the other services will need to be adjusted to `gitHubService: repoService`.

Reply:
okay makes sense